scripts/pylib/twister/twisterlib/statuses.py

```diff
@@ -19,6 +21,21 @@ def _missing_(cls, value):
         if value is None:
             return TwisterStatus.NONE
 
+    @staticmethod
+    def get_color(status: TwisterStatus) -> str:
+        match(status):
```

Comment:
nit: If you use `match` keyword, than Twister requires Python 3.10. Therefore, using `from __future__ import annotations` does not have any sense (it  enables some type annotations for lower versions of python)

Reply:
just mentioned that, because couple days ago @golowanow added PR to still have possibility to run Twister with Python 3.8: https://github.com/zephyrproject-rtos/zephyr/pull/77077
With `match` keyword it is not possible

Reply:
> nit: If you use `match` keyword, than Twister requires Python 3.10. Therefore, using `from __future__ import annotations` does not have any sense (it enables some type annotations for lower versions of python)

You are right - I've used it to change `TwisterStatus` type hint into `Self` - it seems I didn't upstream it. My bad.

When it comes to Python<3.10 support - Zephyr does not claim to support it since 3.7.0 LTS ([Python 3.10 is listed as minimum requirement](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-dependencies)), so I do not pay older versions special consideration.

Reply:
> When it comes to Python<3.10 support - Zephyr does not claim to support it since 3.7.0

the problem is confusing user experience to get SyntaxError just for `twister --help`